x-pack/plugins/alerting/server/application/rule/methods/mute_alert/mute_instance.ts

```diff
@@ -68,15 +77,15 @@ async function muteInstanceWithOCC(
   const mutedInstanceIds = attributes.mutedInstanceIds || [];
   if (!attributes.muteAll && !mutedInstanceIds.includes(alertInstanceId)) {
     mutedInstanceIds.push(alertInstanceId);
-    await context.unsecuredSavedObjectsClient.update(
-      'alert',
+    await muteAlertSo({
```

Comment:
why not just re-using -> `updateRuleSo`, I do not see the advantages to create a specific function for `muteAlertSo`. Am i wrong here?